lib/parser/javascript.js

```diff
@@ -176,18 +166,8 @@ ReplyParser.prototype.execute = function (buffer) {
             } else if (type === 36) { // $
                 ret = this._parseResult(type);
 
-                if (ret === null) {
-                    break;
-                }
-
-                // check the state for what is the result of
-                // a -1, set it back up for a null reply
-                if (ret === undefined) {
```

Comment:
This is dead code by changing the [`return undefined;`] (https://github.com/NodeRedis/node_redis/pull/852/files#diff-778060651e24292442eb47a196897e2eL95) to `return null;`